multigeiger/display.cpp

```diff
@@ -106,28 +106,33 @@ void DisplayGMC(int TimeSec, int RadNSvph, int CPS, bool use_display, bool conne
   if (!isLoraBoard) {
     char output[80];
     int TimeMin = TimeSec / 60;         // calculate number of minutes
-    if (TimeMin >= 999) TimeMin = 999;  // limit minutes to max. 999
+    TimeMin = TimeMin % 999;  // limit minutes to max. 999, roll over
 
     // print the upper line including time and measured radation
+    pu8x8->setFont(u8x8_font_open_iconic_embedded_1x1);
+    sprintf(output, "%c", 0x4A);
+    if (ble_active) pu8x8->print(output);
     pu8x8->setFont(u8x8_font_7x14_1x2_f);
 
-    if (TimeMin >= 1) {                 // >= 1 minute -> display in minutes
-      sprintf(output, "%3d", TimeMin);
+    if (TimeSec < 60)    {    // < 1 minute -> display in seconds
+      sprintf(output, "%2ds", TimeSec);
```

Comment:
good idea. if we have 3 digits for the minutes, 2 digits + "s" for the seconds will also fit.

Reply:
felt more natural :)
Especially as I have wondered a long time what that counter represents. I hope the "s" will give it away more clearly...